hack/make-rules/test-e2e-node.sh

```diff
@@ -138,7 +138,7 @@ if [ $remote = true ] ; then
     --delete-instances="$delete_instances" --test_args="$test_args" --instance-metadata="$metadata" \
     --image-config-file="$image_config_file" --system-spec-name="$system_spec_name" \
     2>&1 | tee -i "${artifacts}/build-log.txt"
-  exit $?
+  exit "${PIPESTATUS[0]}"
```

Comment:
since `hack/lib/init.sh` sets `-o errexit` and `-o pipefail`, I'm not sure we even get to this line.

Reply:
Good to know. Will change back then. :)

Reply:
Done